app/routes.py

```diff
@@ -1,2 +1,53 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, abort, make_response
 
+class Planet:
+    def __init__(self, id, name, description, life):
+        self.id = id
+        self.name = name
+        self.description = description
+        self.life = life
+
+    def to_dict(self):
+        return dict(
+            id=self.id,
+            name=self.name,
+            description=self.description,
+            life=self.life
+        )
+
+
+planets = [
+    Planet(1, "Earth", "Best planet", True),
+    Planet(2, "Saturn", "Got a ring on it", False),
+    Planet(3, "Mars", "We want to go there", False)
+]
+
+bp = Blueprint("planets_bp", __name__, url_prefix="/planets")
+
+
+def validate_planet(id):
```

Comment:
Nice decision to pull this up above the two routes. And the validation method looks good, for both detecting a bad id, as well as an id with no record.